platform-api/che-core-api-builder/src/main/java/org/eclipse/che/api/builder/RemoteBuilder.java

```diff
@@ -106,6 +110,14 @@ public long getLastUsageTime() {
     }
 
     /**
+     * Get Builder Environment map
+     * @return map of BuilderEnvironment
+     */
+    public Map<String, BuilderEnvironment> getBuilderEnvironment(){
+        return this.environmentMap;
```

Comment:
This should be wrapped with Collections.unmodifiableMap(map) to avoid unexpected modifications